docs/src/modules/components/ApiPage.tsx

```diff
@@ -360,6 +360,26 @@ export default function ApiPage(props: ApiPageProps) {
           layoutStorageKey={layoutStorageKey.classes}
           displayClassKeys
         />
+
```

Comment:
Maybe with a h2 "source" + no callout for its content would work great.

Reply:
Yes, looks beter especially with the link in the Table of Content :+1:

![image](https://github.com/user-attachments/assets/d43866fa-47be-49d3-bd0c-7ad1ef56b724)